source/_integrations/motion_blinds.markdown

```diff
@@ -30,3 +30,56 @@ Please note that "-" characters need to be included in the key when providing it
+- 'Absolute position' is the position of the Top bar with respect to the window, so 0 = bottom of the window and 100 = top of the window. Note that not all absolute positions are reachable at all moments due to the Bottom bar.
+- 'Width' is the percentage of the window covered by fabric (the space between the Top and Bottom bars).
+
+### Bottom entity
+
+- 'Up/Open' will move the Bottom bar to the position of the Top bar.
+- 'Down/Close' will move the Top bar to the bottom of the window (absolute position 0)
+- 'Position' is the relative position in which the Bottom bar can move, so from the position of the Top bar (100) to the bottom of the window (0), note that the position will therefore change if the Top bar is moved, since the space in which the Bottom bar is allowed to move changes.
+- 'Absolute position' is the position of the Bottom bar with respect to the window, so 0 = bottom of the window and 100 = top of the window. Note that not all absolute positions are reachable at all moments due to the Top bar.
+- 'Width' is the percentage of the window covered by fabric (the space between the Top and Bottom bars).
+
+### Combined entity
+
+- 'Up/Open' will move both the Top and Bottom bars to the top of the window, effectively covering as little as possible of the window (Width will be 0 %).
+- 'Down/Close' will move the Top bar to the top of the window and the Bottom bar to the bottom of the window, effectively covering the whole window (Width will be 100 %).
+- 'Position' is the relative position of the center between the Bottom and Top bars in which the center can move, so bassically such that the area covered by the Bottom and Top bar can be moved withouth changing its size, such that the Top bar can go to the top of the window and the Bottom bar to the bottom of the window.
+- 'Absolute position' is the position of the center between the Bottom and Top bars with respect to the window, so 0 = bottom of the window and 100 = top of the window. Note that not all absolute positions are reachable at all moments due to the width.
+- 'Width' is the percentage of the window covered by fabric (the space between the Top and Bottom bars).
+
+### TDBU notes
+
+Because the relative position is used by default in Home Assistant, scenes will not work properly with TDBU blinds (depending on the current position of the Top en Bottom, a certain position (70) will correspond to a different position with respect to the window).
+Therefore it is recommanded to use scripts or automations with the TDBU Combined entity that use the `motion_blinds.set_absolute_position` with specifying both the `absolute_position` and `width`.
+That will ensure the same absolute position with respect to the window is achieved withouth letting the Bottom or Top bar move to an absolute_position that is not allowed.
```

Comment:
```suggestion
Because the relative position is used by default in Home Assistant, scenes will not work properly with TDBU blinds (depending on the current position of the Top en Bottom, a certain position (70) will correspond to a different position with respect to the window).

Therefore it is recommended to use scripts or automations with the TDBU Combined entity that use the `motion_blinds.set_absolute_position` with specifying both the `absolute_position` and `width`.

That will ensure the same absolute position with respect to the window is achieved without letting the Bottom or Top bar move to an absolute_position that is not allowed.
```